test/test_helper.js

```diff
@@ -0,0 +1,23 @@
+const request = require('supertest');
+const { app } = require('../src/expressServer.js');
+
+const loginAndReturnCookie = async (credentials) => {
+  // Send a login request and get the signed cookie from the response
+  const resp = await request(app).post('/api/v1/auth/login').send(credentials);
+  // Log the response and cookie header for debugging
+  console.log('Response:', resp);
+  console.log('Cookie Header:', resp.headers['set-cookie']);
```

Comment:
We can remove these :+1:

Reply:
Done 👍, commit bb992665d0a17c0f4f7e5e70f0ee774a172bf987